app/models/spree/subscription.rb

```diff
@@ -127,6 +127,23 @@ def send_prior_notification
       end
     end
 
+    def cancel_stripe_subscription
+      raise "Stripe subscription ID not present" unless stripe_subscription_id.present?
```

Comment:
can you please also append the subscription id in string so its easy to find and we know which data had problem?